src/upper_envelope/fues_numba/fues_numba.py

```diff
@@ -163,11 +178,11 @@ def fast_upper_envelope(
     """
     # TODO: determine locations where endogenous grid points are # noqa: T000
     # equal to the lower bound
-    mask = endog_grid <= lower_bound_wealth
-    if np.any(mask):
-        max_value_lower_bound = np.nanmax(value[mask])
-        mask &= value < max_value_lower_bound
-        value[mask] = np.nan
+    # mask = endog_grid <= tuning_params["lower_bound_wealth"]
+    # if np.any(mask):
+    #     max_value_lower_bound = np.nanmax(value[mask])
+    #     mask &= value < max_value_lower_bound
+    #     value[mask] = np.nan
```

Comment:
Do we need the commented-out part still?

Reply:
Removed. Is mentioned in Issue and we need to discuss with Akshay why he had that.